fixtures/vitest-pool-workers-examples/basics-integration-auxiliary/vitest.config.ts

```diff
@@ -0,0 +1,39 @@
+import { defineWorkersProject } from "@cloudflare/vitest-pool-workers/config";
+
+export default defineWorkersProject({
+	test: {
+		globalSetup: ["./global-setup.ts"],
+		poolOptions: {
+			workers: {
+				singleWorker: true,
+				miniflare: {
+					// Configuration for the test runner Worker
+					compatibilityDate: "2024-01-01",
+					compatibilityFlags: [
+						"nodejs_compat",
+						// Required to use `WORKER.scheduled()`. This is an experimental
+						// compatibility flag, and cannot be enabled in production.
+						"service_binding_extra_handlers",
+					],
+					serviceBindings: {
+						WORKER: "worker",
+					},
+
+					workers: [
+						// Configuration for the "auxiliary" Worker under test.
+						// Unfortunately, auxiliary Workers cannot load their configuration
+						// from `wrangler.toml` files, and must be configured with Miniflare
+						// `WorkerOptions`.
+						{
+							name: "worker",
```

Comment:
Might be helpful to give this a slightly less generic name so that it is obvious that it is being used in the service binding.